src/DynamoCoreWpf/ViewModels/Search/SearchViewModel.cs

```diff
@@ -1089,6 +1089,68 @@ internal void SelectAllCategories(object parameter)
             }
         }
 
+        /// <summary>
```

Comment:
@ramramps , that's what I meant to say... "If we want to set false for all expanded items, we must search for these expanded items".
All in all, LINQ still uses loops inside. But to be sure, I'll check this also... I don't know what is faster foreach loop or LINQ.